build-tools/automation/yaml-templates/run-integrated-regression-tests.yaml

```diff
@@ -40,6 +40,11 @@ steps:
   clean: true
   fetchDepth: 10
 
+  # Clear out VS installer content before provisioning VS to avoid using unstable tools from pre-release installers.
+- powershell: Remove-Item "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer" -Recurse -Force -ErrorAction Ignore
+  displayName: Delete VS Installer
+  condition: and(succeeded(), eq(variables['agent.os'], 'Windows_NT'))
```

Comment:
What puts `C:\Program Files (x86)\Microsoft Visual Studio\Installer` back in place? provisionator?

Will this break things like the Azure DevOps `MSBuild` task? It uses `vswhere.exe` in this directory.

Reply:
The provisionator call which invokes the .csx linked below will regenerate this folder as part of VS provisioning:
https://github.com/xamarin/QualityAssurance/blob/master/Scripts/Provisionator/xamarin-android.csx#L20